docs/ops/convolution/ConvolutionBackpropData_1.md

```diff
@@ -4,15 +4,15 @@
 
 **Category**: Convolution
 
-**Short description**: Computes the gradients of a Convolution operation with respect to the input. Also known as a Deconvolution or a Transposed Convolution.
+**Short description**: Computes 1D, 2D or 3D *ConvolutionBackpropData* operation with respect to the input and kernel tensors. Also known as a Transposed Convolution.
 
 **Detailed description**:
 
 ConvolutionBackpropData takes the input tensor, weights tensor and output shape and computes the output tensor of a given shape. The shape of the output can be specified as an input 1D integer tensor explicitly or determined by other attributes implicitly. If output shape is specified as an explicit input, shape of the output exactly matches the specified size and required amount of padding is computed.
 
-ConvolutionBackpropData accepts the same set of attributes as a regular Convolution operation, but they are interpreted in a "backward way", so they are applied to the output of ConvolutionBackpropData, but not to the input. Refer to a regular Convolution operation for detailed description of each attribute.
+ConvolutionBackpropData accepts the same set of attributes as a regular Convolution operation, but they are interpreted in a "backward way", so they are applied to the output of ConvolutionBackpropData, but not to the input. Refer to a regular [Convolution](Convolution_1.md) operation for detailed description of each attribute.
 
-Output shape when specified as an input `output_shape`, specifies only spatial dimensions. No batch or channel dimension should be passed along with H, W or other spatial dimensions. If `output_shape` is omitted, then `pads_begin`, `pads_end` or `auto_pad` are used to determine output spatial shape `[Y_1, Y_2, ..., Y_D]` by input spatial shape `[X_1, X_2, ..., X_D]` in the following way:
+Output shape when specified as an input `output_shape`, specifies only spatial dimensions. No batch or channel dimension should be passed along with spatial dimensions. If `output_shape` is omitted, then `pads_begin`, `pads_end` or `auto_pad` are used to determine output spatial shape `[Y_z, Y_y, Y_x]` by input spatial shape `[X_z, X_y, X_x]` in the following way:
```

Comment:
Change "Output shape when specified as an input \`output_shape\`, specifies only spatial dimensions." to "When output shape is specified as an input tensor \`output_shape\` then it specifies only spatial dimensions."

Reply:
Names `Y_z, Y_y, Y_x, X_z, X_y, X_x` are really confusing... Maybe rename "X" to "I" and "Y" to "O" to reflect the fact that they correspond to Input and Output tensors?

Reply:
Corrected.